pkg/devfile/adapters/kubernetes/component/adapter.go

```diff
@@ -0,0 +1,88 @@
+package component
+
+import (
+	"fmt"
+
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/golang/glog"
+
+	adapterCommon "github.com/openshift/odo/pkg/devfile/adapters/common"
+	"github.com/openshift/odo/pkg/devfile/versions/common"
+	"github.com/openshift/odo/pkg/kclient"
+)
+
+// New instantiantes a component adapter
+func New(commonAdapter adapterCommon.AdapterContext, client kclient.Client) Adapter {
```

Comment:
Can we rename `commonAdapter adapterCommon.AdapterContext` to `adapterContext common.AdapterContext` so that its the same as kubernetes/adapter.go

Reply:
Yes, good catch. I must have missed that during the refactoring. Done.